dev_guide/deployments/deployment_strategies.adoc

```diff
@@ -433,3 +433,25 @@ pre-deployment hook with the following command:
 $ oc set deployment-hook dc/frontend --pre -c helloworld -e CUSTOM_VAR1=custom_value1 \
   -v data --failure-policy=abort -- /usr/bin/command arg1 arg2
 ----
+
+[[adding-secrets-to-deployment-config]]
+== Adding Secrets to Deployment Configurations from the Web Console
+
+Add a secret to your deployment configuration so that it can access a private
+repository.
+
+. Create a new {product-title} project.
+
+. xref:../../dev_guide/secrets.adoc#dev-guide-secrets[Create a secret] that contains
+credentials for accessing a private source code repository.
```

Comment:
Unlike BC, secret related to DC only includes Pull Secret from private image repository. So here need replace source code repository with image repository